wire-ios/Wire Notification Service Extension/LegacyNotificationService.swift

```diff
@@ -141,6 +143,19 @@ public final class LegacyNotificationService: UNNotificationServiceExtension, No
         contentHandler(content)
     }
 
+    private func removeNotification(withSameMessageId messageNonce: UUID?) {
```

Comment:
this will remove any notification with the same message id already delivered, maybe during another fetch of notifcations stream.

This will be also useful for another upcoming PR to replace edited messages